janus/src/message.rs

```diff
@@ -689,6 +689,9 @@ pub struct HpkeConfig {
 }
 
 impl HpkeConfig {
+    /// The media type associated with this protocol message.
+    pub const MEDIA_TYPE: &'static str = "application/dap-hpke-config";
```

Comment:
I think you're right on the money. Since this PR is blocked on resolving https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/issues/260 anyway, should we block it on resolving the change to update the `CONTENT_TYPE`, too?